mocha/test/version.test.js

```diff
@@ -0,0 +1,27 @@
+const httpStatus= require("http-status");
+const APP_URL = process.env.MOCHA_URL;
+const apiEndPoints = require("../utils/apiEndpoints");
+const apiHelper = require("../utils/apiHelper");
+const validator = require("../utils/validators");
+const testData = require("../utils/testData");
+
+let apiEndPoint= apiEndPoints.versionApiEndpoint;
+
+describe("Get Version endpoints test", function(){
+    it("user enters to see version",async function(){
+            let response= await apiHelper.sendPOSTRequest(
+            APP_URL,
+            apiEndPoint
+    );
+    validator.validateStatusCodeAndOk(
+        response,
```

Comment:
Formatting - do you have Prettier installed for your IDE like VSC?
If not look here: https://github.com/FightPandemics/FightPandemics/pull/1717#issuecomment-719582599

Reply:
Thank you, Agnieszka. I don't have Prettier but I will install it.